src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php

```diff
@@ -404,6 +406,29 @@ protected function createFormBuilder($data = null, array $options = array())
         return $this->container->get('form.factory')->createBuilder(FormType::class, $data, $options);
     }
 
+    /**
+     * Handles request and check form validity.
+     *
+     * @param FormInterface $form    The form to validate
+     * @param Request|null  $request The request
+     *
+     * @return bool
+     *
+     * @final since version 4.1
+     */
+    protected function isFormValid(FormInterface $form, Request $request = null)
```

Comment:
In order to deduce what this internally does, what about naming it: `isSubmittedFormValid()`?